.copier-answers.yml

```diff
@@ -0,0 +1,17 @@
+# Do NOT update manually; changes here will be overwritten by Copier
+_commit: v1.1.1
+_src_path: https://github.com/OCA/oca-addons-repo-template.git
+dependency_installation_mode: OCA
+generate_requirements_txt: true
+include_wkhtmltopdf: true
+odoo_version: 13.0
+rebel_module_groups: []
+repo_description:
+  Addons for the management of multi company instances as well as inter company
+  communication.
+repo_name: Multi Company
+repo_slug: multi-company
+travis_apt_packages:
+  - expect-dev
+  - python-lxml
```

Comment:
Ah and we probably don't need these two. At least `lxml` is not needed because it was an optimization, but nowadays there is a pre-built wheel on PyPI and if not the pip cache will be sufficient for performance.

Reply:
Ok, I kept them because they were in the old travis.yaml. Should I leave `expected-dev` only?

Reply:
I just checked and [expect-dev is installed always](https://github.com/OCA/oca-addons-repo-template/blob/416ab75ccf656fbdde261fa9846903266bf90b3a/.travis.yml.jinja#L22), so you can leave it out.